dynamoose.d.ts

```diff
@@ -16,6 +16,18 @@ declare module "dynamoose" {
   export function setDefaults(options: ModelOption): void;
   export function setDDB(ddb: _AWS.DynamoDB): void;
   export function revertDDB(): void;
+  export function transaction<DataSchema, KeySchema>(
+    items: Array<
+      Promise<ModelSchema<DataSchema>>
```

Comment:
I don't think this is correct. `dynamoose.transaction` doesn't accept a promise in the array.

Reply:
Reference line 222, 241 in lib/index.js.
First paramater items of dynamoose.transaction is used await Promise.all(items);